ledger/src/shred/merkle.rs

```diff
@@ -70,7 +70,7 @@ pub struct ShredCode {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq)]
-pub(super) enum Shred {
```

Comment:
Is it 100% required to make this enum public? We wanted to get rid of Data shreds at some point, having this stuck in public API will be a major setback.

Reply:
Actually I don't need the `Shred` enum anymore, I can remove that change.